lib/build/setup-scripts/frontend/supertokens-react-custom.js

```diff
@@ -0,0 +1,30 @@
+import path from "path";
```

Comment:
this file is not there in the ts folder? Maybe this needs to go?

Reply:
Good catch, I totally missed this out. I have removed it. Thanks